adam_preprocessing/shape_stroke_extraction.py

```diff
@@ -502,6 +530,15 @@ def get_strokes(self):
                         strokes_normalized_coordinates=reduced_strokes_norm.transpose(
                             0, 2, 1
                         ).tolist(),
+                        size=dict(
+                            # FIXME: calling .item() causes crash (it says they
+                            #  are type `float`), but if they aren't cast to
+                            #  `float`, they get serialized as complex objects,
+                            #  not primitive floats.
```

Comment:
🤔

Reply:
to clarify, not a problem with your code -- I am just wondering to myself why this happens